pkg/actions/irsa/create.go

```diff
@@ -9,7 +9,7 @@ func (a *Manager) CreateIAMServiceAccount(iamServiceAccounts []*api.ClusterIAMSe
 	taskTree := a.stackManager.NewTasksToCreateIAMServiceAccounts(iamServiceAccounts, a.oidcManager, kubernetes.NewCachedClientSet(a.clientSet))
 	taskTree.PlanMode = plan
 
-	err := doTasks(taskTree)
+	err := doTasks(taskTree, "create")
```

Comment:
Can we create enums for "create", "delete", "update" and reuse them in `doTasks` call?